spmd/tensor/ops/tensor_ops.py

```diff
@@ -39,6 +39,7 @@ def no_shard_prop_rule(op_schema: OpSchema) -> OutputSharding:
     "aten.is_same_size.default",
     "aten.ones_like.default",
     "aten.new_empty_strided.default",
+    "aten._softmax_backward_data.default",
```

Comment:
Curious, is this enough for it to work? We might want to give it a prop rule. Because down the road, if we want to add sharding dim softmax, we need to call collectives in the backward as well.

Reply:
Right. I currently leave it as a default rule and it kind of works on `y.backward()` but not on `y.sum().backward()` as I mentioned earlier. Need investigate what is missing.